src/quo2/components/tags/number_tag/style.cljs

```diff
@@ -55,10 +55,11 @@
     (get-in sizes [size (if widen? :width-extra :size)])))
 
 (defn container
-  [{:keys [type number size blur? theme]}]
-  {:style {:width            (get-width size number)
-           :height           (get-in sizes [size :size])
-           :border-radius    (get-shape-value size :border-radius type)
-           :justify-content  :center
-           :align-items      :center
-           :background-color (get-bg-color blur? theme)}})
+  [{:keys [type number size blur? theme container-style]}]
+  {:style (merge {:width            (get-width size number)
+                  :height           (get-in sizes [size :size])
+                  :border-radius    (get-shape-value size :border-radius type)
+                  :justify-content  :center
+                  :align-items      :center
+                  :background-color (get-bg-color blur? theme)}
+                 container-style)})
```

Comment:
`container-style` could override the `width` and `height` of the component, and maybe we don't want it.
So, wdyt about:
`(assoc container-style` :width ... :height ... :border-radius ... )` to preserve the original styles even if they try to override them?

Reply:
Agreed 👍